cores/arduino/pins_arduino_digital.h

```diff
@@ -535,4 +535,552 @@ enum {
   #error "Core NUM_DIGITAL_PINS limited to 192"
 #endif
 
+/*
+ * Each STM32 pin number PYn defined in the variant.h
+ * is also the alternate pin number PYn_ALT0
+ */
+#ifdef PA0
+  #define PA0_ALT0       (PA0  | ALT0)
+#endif
+#ifdef PA1
+  #define PA1_ALT0       (PA1  | ALT0)
+#endif
+#ifdef PA2
+  #define PA2_ALT0       (PA2  | ALT0)
+#endif
+#ifdef PA3
+  #define PA3_ALT0       (PA3  | ALT0)
+#endif
+#ifdef PA4
+  #define PA4_ALT0       (PA4  | ALT0)
+#endif
+#ifdef PA5
+  #define PA5_ALT0       (PA5  | ALT0)
+#endif
+#ifdef PA6
+  #define PA6_ALT0       (PA6  | ALT0)
+#endif
+#ifdef PA7
+  #define PA7_ALT0       (PA7  | ALT0)
+#endif
+#ifdef PA8
+  #define PA8_ALT0       (PA8  | ALT0)
+#endif
+#ifdef PA9
+  #define PA9_ALT0       (PA9  | ALT0)
+#endif
+#ifdef PA10
+  #define PA10_ALT0      (PA10 | ALT0)
+#endif
+#ifdef PA11
+  #define PA11_ALT0      (PA11 | ALT0)
+#endif
+#ifdef PA12
+  #define PA12_ALT0      (PA12 | ALT0)
+#endif
+#ifdef PA13
+  #define PA13_ALT0      (PA13 | ALT0)
+#endif
+#ifdef PA14
+  #define PA14_ALT0      (PA14 | ALT0)
+#endif
+#ifdef PA15
+  #define PA15_ALT0      (PA15 | ALT0)
+#endif
+#ifdef PB1
```

Comment:
I looks like PB0_ALT0 is missing.
The same for all Py0_ALT0 (except for PA0_ALT0)

Reply:
Right. Added.